src/WebEid.Security/Util/X509CertificateExtensions.cs

```diff
@@ -69,6 +69,8 @@ public static void ValidateCertificateExpiry(this Org.BouncyCastle.X509.X509Cert
 
         public static X509Certificate2 ValidateIsSignedByTrustedCa(this X509Certificate2 certificate, ICollection<X509Certificate2> trustedCaCertificates)
```

Comment:
I would like to make the method name intention revealing:
```suggestion
        public static X509Certificate2 ValidateIsValidAndSignedByTrustedCa(this X509Certificate2 certificate, ICollection<X509Certificate2> trustedCaCertificates)
```
I know this is missing in Java, but I plan to change it.

Reply:
I renamed the method with the latest commit.